psync-fconsumer/full-consumer.c

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
I would suggest you make psync as a module under /app-support, leave some APIs available to developers.
The consumer example can be placed under ndn-lite-Posix repo.

But I am fine with the current ongoing code, we can do the aforementioned things later when codes are ready.